packages/cli/src/__tests__/external-hooks.test.ts

```diff
@@ -1,12 +1,12 @@
 import type { GlobalConfig } from '@n8n/config';
 import type { WorkflowRepository } from '@n8n/db';
 import type { CredentialsRepository, SettingsRepository } from '@n8n/db';
+import type { UserRepository } from '@n8n/db';
```

Comment:
Imports were automatically lintfixed, so they are be grouped in all cases, but it's not a good use of time to fix imports manually. 

If we really want better grouping, we'd have to take over the the lint rule to handle more edge cases.